test/admin/service.test.js

```diff
@@ -0,0 +1,74 @@
+const StitchMongoFixture = require('../fixtures/stitch_mongo_fixture');
```

Comment:
[req] Can you add a few failure tests for:
* Invalid service name
* Unknown service type
* Invalid config

Reply:
✅